src/store/cart/saga.ts

```diff
@@ -1,11 +1,99 @@
+import { addToCart, removeItemsFromCart, updateItemQuantities } from '@graphql/mutation'
 import { call, takeLatest } from '@redux-saga/core/effects'
-import { showDialog } from '@utils'
-import { ADDED_ITEM } from './action-types'
+import { SHOW_LOADER, SHOW_TOAST } from '@store/intermitence/action-types'
+import { SIGN_UP_ASYNC } from '@store/user/action-types'
+import { actionObject, filter, GraphQlClient, manageError, reduceVariation, showDialog, validateFetch } from '@utils'
+import { put, select } from 'redux-saga/effects'
+import { ADDED_ITEM, REMOVE_ITEM, SET_ITEM, UPDATE_QUANTITY } from './action-types'
 
-function* addedItemAsync() {
-  yield call(showDialog, 'Producto agregado al carrito')
+function* addedItemAsync({ payload }: any) {
+  try {
+    const { user: { user: { sessionToken }, isAuth } } = yield select(state => state)
```

Comment:
For this case, create a selector, for example:

In selectors.ts: export const getUser = (state: any) => state.user
In cart saga: const { user: { user: { sessionToken }, isAuth } } = yield select(getUser)